qiskit/pulse/instructions/instruction.py

```diff
@@ -77,6 +78,11 @@ def __init__(self,
             raise PulseError("{} duration of {} is invalid: must be nonnegative."
                              "".format(self.__class__.__name__, duration))
         self._duration = duration
+
+        for channel in channels:
+            if not isinstance(channel, Channel):
+                raise PulseError('Input {} is not a channel.'.format(channel))
```

Comment:
Maybe rephrase this from the perspective of helping the user fix their issue ("Expected a channel, got {} instead.")? For my understanding, where does this come up from the builder?

Reply:
Fixed error message in 8625c29e0.